### PR TITLE
[2.6] Rebase misc

### PR DESCRIPTION
**Pull request by yannuil**

### CODE REVIEW

File: bin/update_maistra_deps.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"maistra-2.4"}
```
Comment (yxun) [inline bin/update_maistra_deps.sh:19]:
```suggestion
UPDATE_BRANCH=${UPDATE_BRANCH:-"maistra-2.6"}
```

**Comment by jewertow** [comment]
It seems that you forgot to run `make gen`.

**Review by jewertow — APPROVED** [review]

**Pull request merged**